packages/tax/src/services/tax-module-service.ts

```diff
@@ -246,12 +227,20 @@ export default class TaxModuleService<
       {
         $or: [
           {
-            country_code: calculationContext.address.country_code,
+            country_code: this.normalizeRegionCodes(
```

Comment:
A nicer approach is to have a single `normalizeTaxLinesCalculationContext` function that will normalize all input at the beginning of the method, and then you don't need to worry about forgetting to normalize in one place anymore.

Reply:
Great push! Thanks